.github/workflows/ci-check.yml

```diff
@@ -10,24 +10,80 @@ concurrency:
   group: ci-check-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  NODE_VERSION: "14.x"
+
 jobs:
-  ci-check:
+  format:
+    name: Format
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [14.x]
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js
-        uses: actions/setup-node@v2
+      - uses: actions/setup-node@v2
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --offline
+      - run: sudo apt-get install xvfb
+      - run: xvfb-run --auto-servernum yarn format:diff
```

Comment:
@jeffchew why the `xvfb-run --auto-servernum`? Can we remove it?

Reply:
Not sure, this was originally put in by @asudoh .

Reply:
@asudoh could you confirm us? Thanks!

Reply:
@abdonrd @jeffchew `xvfb-run` in this file is for ensuring GUI code (of browsers) runs in CI, notably for Karma, Puppeteer and Playwright. You can see the the code back in April for your reference: https://github.com/carbon-design-system/carbon-web-components/blob/v1.13.0/.github/workflows/ci-check.yml

Reply:
Ok, thanks Akira!